google-cloud-bigtable/pom.xml

```diff
@@ -545,7 +549,36 @@
   </profiles>
 
   <build>
+    <extensions>
```

Comment:
conformance test is added as a dependency: https://github.com/googleapis/java-bigtable/blob/main/google-cloud-bigtable/pom.xml#L210-L228 and I think the test proto is defined here: https://github.com/googleapis/conformance-tests/blob/ec2e8c3d312e7122e07f65e98639169515449aa8/bigtable/v2/proto/google/cloud/conformance/bigtable/v2/tests.proto

Can you make the changes in conformance test so we don't need to add these plugins? and the proto will get generated from gapic. 

@igorbernstein2  what do you think?

Reply:
Discussed offline with Igor. We can keep it in the google-cloud-bigtable/ for now since we don't have plan to support this feature outside of JAVA. If in the future we plan to have cross-language support, we should move them to the conformance test.